mlir/lib/Conversion/VectorToLLVM/ConvertVectorToLLVM.cpp

```diff
@@ -1937,13 +2000,14 @@ void mlir::populateVectorToLLVMConversionPatterns(
                VectorExtractElementOpConversion, VectorExtractOpConversion,
                VectorFMAOp1DConversion, VectorInsertElementOpConversion,
                VectorInsertOpConversion, VectorPrintOpConversion,
-               VectorTypeCastOpConversion, VectorScaleOpConversion,
-               VectorLoadStoreConversion<vector::LoadOp>,
+               VectorTypeCastOpConversion, VectorScaleOpConversion>(converter);
+  patterns.add<VectorLoadStoreConversion<vector::LoadOp>,
                VectorLoadStoreConversion<vector::MaskedLoadOp>,
                VectorLoadStoreConversion<vector::StoreOp>,
                VectorLoadStoreConversion<vector::MaskedStoreOp>,
-               VectorGatherOpConversion, VectorScatterOpConversion,
-               VectorExpandLoadOpConversion, VectorCompressStoreOpConversion,
+               VectorGatherOpConversion, VectorScatterOpConversion>(
+      converter, useVectorAlignment);
+  patterns.add<VectorExpandLoadOpConversion, VectorCompressStoreOpConversion,
```

Comment:
Wouldn't one additional `patterns.add` be sufficient?

Reply:
Yes, I think so, since it's a pattern set and thus the order of the patterns doesn't matter